packages/cli/src/commands/epochs/switch.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { CustomFlags } from '../../utils/command'
+
+export default class Switch extends BaseCommand {
+  static description = 'Finishes current epoch and starts a new one.'
+
+  static flags = {
+    ...BaseCommand.flags,
+    // TODO not sure if this flag is needed at all here
+    from: CustomFlags.address({ required: true }),
+  }
+
+  static args = {}
+
+  static examples = ['switch --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95']
+
+  async run() {
+    const kit = await this.getKit()
+    const res = await this.parse(Switch)
+    const address = res.flags.from
+
+    kit.defaultAccount = address
+
+    const epochManager = await kit.contracts.getEpochManager()
+
+    await displaySendTx('startNextEpoch', epochManager.startNextEpochProcess())
```

Comment:
is there a reason the awaits are not symetrical here?

start does not have one finish does. 


is this do reduce the time between the calls?

Reply:
yes

`startNextEpochProcess` returns a tx object directly (proxy to contract call)

`finishNextEpochProcessTx` is actually a function that contains logic in order to prepare the actual tx object and since it does async calls, it needs to be awaited, but in the end it results a txobject like the call above

IMHO this function should be called `prepareFinishNextEpochProcessTx` to be more verbose, WDYT @pahor167 ?